### ISSUE TITLE
Issue with chefdk 0.11.0

### ISSUE THREAD

**Issue by mrdimka** [chef/knife-reporting#15]
I have recently upgraded chefdk from version 0.10.0 to 0.11.0
Everything worked perfect till now but after the upgrade when trying to execute knife runs list 

<pre>
 $ knife runs list myNode -VV
INFO: Using configuration from /home/dimka/Projects/MyCHEF/.chef/knife.rb
/home/dimka/.chefdk/gem/ruby/2.1.0/gems/knife-reporting-0.4.1/lib/chef/knife/runs_list.rb:65:in `run': uninitialized constant Chef::REST (NameError)
    from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.7.2/lib/chef/knife.rb:416:in `block in run_with_pretty_exceptions'
    from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.7.2/lib/chef/local_mode.rb:44:in `with_server_connectivity'
    from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.7.2/lib/chef/knife.rb:415:in `run_with_pretty_exceptions'
    from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.7.2/lib/chef/knife.rb:214:in `run'
    from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.7.2/lib/chef/application/knife.rb:148:in `run'
    from /opt/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.7.2/bin/knife:25:in `<top (required)>'
    from /usr/bin/knife:50:in `load'
    from /usr/bin/knife:50:in `<main>'
</pre>

**Comment by msnahill** [comment]
I can confirm that I am experiencing the same issue.  I installed _chefdk-0.11.2_ (Windows) and the `knife runs list` command throws the following exception:
```
D:\gitrepo>knife runs list servername.domainname.com -VV
INFO: Using configuration from D:/gitrepo/.chef/knife.rb
C:/Users/username/AppData/Local/chefdk/gem/ruby/2.1.0/gems/knife-reporting-0.4.1/lib/chef/knife/runs_list.rb:65:in `run': uninitialized constant Chef::REST (NameError)
        from C:/opscode/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.7.2-universal-mingw32/lib/chef/knife.rb:416:in`block in run_with_pretty_exceptions'
        from C:/opscode/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.7.2-universal-mingw32/lib/chef/local_mode.rb:44:in `with_server_connectivity'
        from C:/opscode/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.7.2-universal-mingw32/lib/chef/knife.rb:415:in`run_with_pretty_exceptions'
        from C:/opscode/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.7.2-universal-mingw32/lib/chef/knife.rb:214:in`run'
        from C:/opscode/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.7.2-universal-mingw32/lib/chef/application/knife.rb:148:in `run'
        from C:/opscode/chefdk/embedded/lib/ruby/gems/2.1.0/gems/chef-12.7.2-universal-mingw32/bin/knife:25:in `<top (required)>'
        from C:/opscode/chefdk/bin/knife:60:in `load'
        from C:/opscode/chefdk/bin/knife:60:in `<main>'
```

**Comment by simxitnsz** [comment]
I'm running windows as well and I made the changes suggested in this thread and it seems to be working for me: https://github.com/chef/chef/commit/9bb3f00df9774d352e277fa9f03e7c2009e00598

**Comment by chrisgit** [comment]
@simxitnsz Thanks for the link and advice, the code change needs to be applied to runs_show and runs_list but works fine for me.

**Comment by tas50** [comment]
This was fixed in 0.5.0
